docs/update_last_modified.sh

```diff
@@ -0,0 +1,17 @@
+#!/opt/homebrew/bin/bash
```

Comment:
Just use basic bash here
```suggestion
#!/bin/bash
```

Reply:
Also move this script to a dir named `bin` to match all our other repos plz

Reply:
Done.